innopoints/views/product.py

```diff
@@ -32,10 +32,11 @@ def list_products():
     """List products available in InnoStore."""
     default_limit = 3
     default_page = 1
-    default_order = 'time'
+    default_order = 'name'
     ordering = {
-        'time': Product.addition_time,
-        'price': Product.price
+        'name': Product.name.asc(),
+        'price_asc': Product.price.asc(),
+        'price_desc': Product.price.desc()
```

Comment:
why did you remove ordering by time? It would help identify the new products.
and I would say ordering default to ascending, but why exclusively? I would suggest to rename the param to `order_by` and add `order=asc|desc` (actually, `order_by` as the param name makes more sense for the criteria even without specifying asc/desc)